test/unit/components/chart/header.test.js

```diff
@@ -0,0 +1,295 @@
+/* global chai */
+
+var React = require('react');
+var TestUtils = require('react/lib/ReactTestUtils');
+var _ = require('lodash');
+var expect = chai.expect;
+var rewire = require('rewire');
+var rewireModule = require('../../../utils/rewireModule');
+
+describe('Header', function () {
+  var Header = rewire('../../../../app/components/chart/header');
```

Comment:
Are you referring to rewire? It does seem quite heavy, but it really does help with isolation when running unit tests with this coding style.